### PR TITLE
[ticket/13437] Add template event viewtopic_body_post_author_before/after

### PR DESCRIPTION
**Pull request by kinerity**
PHPBB3-13437

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1455,6 +1455,22 @@ viewtopic_body_poll_question_prepend
 * Since: 3.1.0-b3
 * Purpose: Add content directly before the poll question on the View topic screen
 
+viewtopic_body_post_author_after
+===
+* Locations:
+    + styles/prosilver/template/viewtopic_body.html
+    + styles/subsilver2/template/viewtopic_body.html
+* Since: 3.1.3-RC1
+* Purpose: Add content directly after the post author on the View topic screen
```
Comment (nickvergessen) [inline phpBB/docs/events.md]: view should be lower case, same below

Comment (kinerity) [inline phpBB/docs/events.md]: I copied that from the above event line (viewtopic_body_poll_question_prepend). Should I still change it?

Comment (nickvergessen) [inline phpBB/docs/events.md]: yeah, change it here (we dont need to copy typos ;) )

**Comment by nickvergessen** [comment]
Can you quickly explain, what you plan to do with these events?
It looks like you want to wrap the username with some HTML?
This is potentially dangerous, since another extension doing the same thing will cause the page to be invalid html

**Comment by kinerity** [comment]
This is just an example, but the best I could come up with at the moment.

Assume an extension that wants to indicate a user as the original poster of the topic. WIth these events, you can place the code before or after the username (I included both even though I would only use the "after" event, as it seems standard prac). I could then create my template event file like so (prosilver example):

<div class="original_poster">{L_ORIGINAL_POSTER}</div>

My intention is NOT to wrapt the username in HTML (I'm a little smarter than that). *wink*

**Pull request merged**